sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -977,7 +977,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     Azure::Nullable<std::string> Id;
 
     /**
-     * @brief Organization Administators collection.
```

Comment:
do we know why cspell didn't catch this and the other error?

Reply:
I asked myself the same question. This file was added before cspell was activated across the repo (1+ year ago), and hadn't been modified since, so it never got flagged.